serving/serving.go

```diff
@@ -82,13 +79,182 @@ func (serv *FeatureServer) TrainingData(req *pb.TrainingDataRequest, stream pb.F
 	return nil
 }
 
+type splitContext struct {
```

Comment:
what is the convention?

Reply:
I mostly agree and thought about it a bit too but with it being an internal struct and literally being the context for all the functions I thought it would do.